tests/rules/test_json.py

```diff
@@ -0,0 +1,21 @@
+from validator.rules import JSON
+
+
+def test_json_01():
+    assert JSON().check("{}")
+
+    assert JSON().check('{ "age":100}')
+
+    assert JSON().check('{"age":100 }')
+
+    assert JSON().check('{"age":100 }')
```

Comment:
Great work! 

JSON is complex and can handle more than one variable in array or dict. Please add more complex tests